Orange/data/io.py

```diff
@@ -319,7 +320,7 @@ def _get_active_sheet(self) -> openpyxl.worksheet.worksheet.Worksheet:
             return self.workbook.active
 
     @classmethod
-    def write_file(cls, filename, data):
+    def write_file(cls, filename, data, with_annotations=True):
```

Comment:
Also this changes the behavior. I'd set it to False. My reasoning is that users who would not want to re-import the edited file back into Orange, would actually prefer it to be without annotations, which only make sense for Orange readers. :)

Reply:
Fixed